alloydb-jdbc-connector/pom.xml

```diff
@@ -124,6 +122,14 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
```

Comment:
you instead want to import this as part of `dependencyManagement`. See https://github.com/googleapis/java-datastore/blob/main/pom.xml#L149-L158 as an example.

Reply:
updated. Thanks!